templates/test/ci/patches/control-plane-ci-version.yaml

```diff
@@ -20,14 +20,7 @@ spec:
         set -o pipefail
         set -o errexit
         [[ $(id -u) != 0 ]] && SUDO="sudo" || SUDO=""
-        GSUTIL=gsutil
```

Comment:
please also change `CI_URL="gs://kubernetes-release/release/$$CI_VERSION/bin/linux/amd64"` on line 47 to `CI_URL="https://storage.googleapis.com/kubernetes-release/release/$${KUBERNETES_VERSION}/bin/linux/amd64"`

Reply:
Is it for every file?

Reply:
yes